documentation/README.md

```diff
@@ -177,7 +178,7 @@ const assign = require('1-liners/assign');
 
 const yedi = { id: 1, age: 100 };
 
-assign({ name: 'Yoda', age: 900 }, yedi);  // => { id: 1, age: 900, name: 'Yoda' }
+assign({ name: 'Yoda', age: 900 }, yedi);  // => { id: 1, name: 'Yoda', 900 }
```

Comment:
The file was automatically generated.
This is in the master: https://github.com/1-liners/1-liners/blob/master/module/assign.js#L14

Should I fix it in this PR?